### PR TITLE
Improve Sparse Vectors Readability

### PR DESCRIPTION
**Pull request by NirantK**

### CODE REVIEW

File: qdrant-landing/content/documentation/concepts/indexing.md [hunk 1]
```diff
@@ -228,32 +228,31 @@ The HNSW parameters can also be configured on a collection and named vector
 level by setting [`hnsw_config`](../indexing/#vector-index) to fine-tune search
 performance.
 
-## Sparse vector index
+## Sparse Vector Index
 
-*Available as of v1.7.0*
+**Version Availability:** Available as of v1.7.0
```
Comment (agourlay) [inline qdrant-landing/content/documentation/concepts/indexing.md:233]: I think we have the format `*Available as of Vx` all over the docs.
Why changing this one?

Comment (NirantK) [reply qdrant-landing/content/documentation/concepts/indexing.md:233]: The existing one is quite hard to read and people have missed it in the past on GH and Discord. I want to make a stab at making it more visible and harder to miss. 

If you think this is the wrong place for this experiment, let's keep it as we have in rest of the docs.

Comment (agourlay) [reply qdrant-landing/content/documentation/concepts/indexing.md]: This feedback is an interesting data point we should consider for a refactoring :+1:

**Comment by agourlay** [comment]
sparse-doc

IMHO it looks a bit crowded with the bold subtitle plus a single description line.

File: qdrant-landing/content/documentation/concepts/search.md [hunk 2]
```diff
@@ -258,9 +258,12 @@ If the collection was created with sparse vectors, the name of the sparse vector
 You can still use payload filtering and other features of the search API with sparse vectors.
 
 There are however important differences between dense and sparse vector search:
-- only `Dot` metric is supported for sparse vectors (no need to specify it in the request)
-- the sparse search is not approximate, it is always returning the exact match
-- the spearse search returns only the vectors which have non-zero values in the same indices as the query vector, for this reason, you can can receive less than `limit` results.
+
```
Comment (agourlay) [inline qdrant-landing/content/documentation/concepts/search.md:261]: I like this change :+1:

**Review by timvisee — APPROVED** [review]

**Pull request merged**